shared/dates.js

```diff
@@ -1,19 +1,24 @@
+// TODO: momentjs is no longer supported. Replace with another date utility/practice. See https://momentjs.com/docs/#/-project-status/.
```

Comment:
Very low priority item I had done several years ago in #483 and willing to redo if others feel strongly for this, in a separate PR.